src/ContentRepository90/Rules/FusionNodeHiddenInIndexRector.php

```diff
@@ -30,13 +30,10 @@ public function refactorFileContent(string $fileContent): string
                 '// TODO 9.0 migration: Line %LINE: You may need to rewrite "VARIABLE.hiddenInIndex" to VARIABLE.property(\'hiddenInIndex\'). We did not auto-apply this migration because we cannot be sure whether the variable is a Node.'
             )
             ->process(fn(string $eelExpression) => preg_replace(
-                '/([a-zA-Z.]+(node|documentNode))\.property\\(\'_hiddenInIndex\'\\)/',
-                '$1.property(\'hiddenInIndex\')',
+                '/\.property\\((\'|")_hiddenInIndex(\'|")\\)/',
+                '.property(\'hiddenInIndex\')',
                 $eelExpression
-            ))
-            ->addCommentsIfRegexMatches(
-                '/\.property\\(\'_hiddenInIndex\'\\)/',
-                '// TODO 9.0 migration: Line %LINE: You may need to rewrite "VARIABLE.property(\'_hiddenInIndex\')" to VARIABLE.property(\'hiddenInIndex\'). We did not auto-apply this migration because we cannot be sure whether the variable is a Node.'
```

Comment:
i guess the assumption is fair here to rewrite `blabla.property('_hiddenInIndex\')` as well?

Reply:
This doesn't make sence anymore, as the regex above is matching this already